randovania/games/prime2/pickup_database/pickup-database.json

```diff
@@ -602,7 +602,7 @@
                 "Grapple",
                 "ScrewAttack"
             ],
-            "default_shuffled_count": 0,
+            "default_shuffled_count": 2,
```

Comment:
This is the progressive item (Grapple, Screw).
You now have shuffled grapple, shuffled screw and 2 shuffled (grapple, screw) progression?

Reply:
as far as I understand it, the "default shuffled" is what's being used as the default if "shuffled" is selected. *Not* whether this progressive is enabled by default.
See dread's progressive suits for example. Gravity suit is default shuffled 1, varia suit is default shuffled 1 and progressive suit is default shuffled 2, because when you use progressives, you want 2 in the world (one for varia, one for gravity)

Reply:
Oh yeah, that sounds like a good reason to stop reviewing code over 30 °C. My brain associated it with the starter preset and I have no idea why :).
If you use progressive it really should default to the amount you need to get all stages of the progression. 👍